### PR TITLE
Feature/query service

### PR DESCRIPTION
**Pull request by motxx**
## What is this pull request?
Implementation of torii QueryService

## Why do you implement it? Why do we need this pull request?
Server for queries from clients.

## How to use the features provided in the pull request?
Use `ServerRunner::run()` then QueryServer starts.

## Details/Features
List of features / major commits
- Add torii/query_server.hpp/cpp
- Add QueryServer to ServerRunner
- Add connection test of QueryServer

### CODE REVIEW

File: irohad/ordering/CMakeLists.txt [hunk 1]
```diff
@@ -14,15 +14,15 @@
...
+#add_library(ordering STATIC queue.cpp observer.cpp)
 
-target_link_libraries(ordering
-  peer_service
-  command_service
-  ordering_service
-  consensus_service
-  ordering_connection
-)
+#target_link_libraries(ordering
```
Comment (grimadas) [inline irohad/ordering/CMakeLists.txt]: Maybe you can just delete it

Comment (motxx) [inline irohad/ordering/CMakeLists.txt]: ok, I'll change.

**Comment by motxx** [comment]
I found some bug as following:

**Comment by motxx** [comment]
The bug is fixed by merging event-loop about CommandService and QueryService that is using same port.
Now, QueryService is *async*. If clients want to use sync, they should request with async and block their thread.

If we have to make QueryService *grpc sync only*, maybe ports of CommandService and QueryService have to be different (I don't know).

And it's better that we'll create a test that runs many processes (clients) and one server, then calculates how many requests can be resolved.

**Pull request merged**